tools/stress-cluster/chaos/README.md

```diff
@@ -0,0 +1,201 @@
+azure-sdk-chaos is an environment and set of tools for performing long-running tests with the Azure SDK that involve OS and system-level faults.
```

Comment:
Should this be under doc directory instead of tools? Or do you plan to add some sort of source code at some point?

Reply:
Yeah I'm planning on making this the initial home of all buildout automation, etc. in addition to the current docs.